build/MicroBuild.Plugins.props

```diff
@@ -0,0 +1,27 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This comes from the Microbuild package, right? What's the story on including it directly?

Reply:
The .vsmanproj doesn't really support the PackageReference to MicroBuild (at least, I couldn't cajole it into working). So, in order to support building locally (which for that project requires the Swix plugin), you'd need to download the MicroBulid Swix package, and then set MicroBuildOverridePluginDirectory to point to where that is.  It's not elegant, but it works.

I don't know if the rest of this file will ever be used aside from the last override, but I reproduced it here just in case.

Reply:
(I guess that's not limited to building locally, the CI needs to have this target too.  But it already knows to install the plugin and set the environment variable accordingly.)